lighthouse-core/gather/driver/execution-context.js

```diff
@@ -94,7 +94,7 @@ class ExecutionContext {
         return new Promise(function (resolve) {
           return Promise.resolve()
             .then(_ => ${expression})
-            .catch(${pageFunctions.wrapRuntimeEvalErrorInBrowserString})
+            .catch(${pageFunctions.wrapRuntimeEvalErrorInBrowser.toString()})
```

Comment:
why call toString explicitly when all objects in a template string are coerced via `.toString()` automatically?